themes/classic/templates/checkout/_partials/cart-summary.tpl

```diff
@@ -54,7 +54,7 @@
 
     {block name='cart_summary_subtotals'}
       {foreach from=$cart.subtotals item="subtotal"}
-        {if $subtotal && $subtotal.type !== 'tax'}
+        {if $subtotal.value && $subtotal.type !== 'tax'}
```

Comment:
This line might have side-effects. If `$subtotal.value` is 0 (integer), it will hide the line instead of display it. This could create issue if the merchant wants to display some subtotal to the customer for legal compliance matters 🤔 ?

Reply:
I say this because the change will impact every subtotals passed in `$cart.subtotals`, not only the shipping subtotal 😉